configure

```diff
@@ -522,6 +521,13 @@ while true; do
   CUDA_TOOLKIT_PATH=""
 done
 
+# Set default CUDA version if not set
+if [ -z "$TF_CUDA_VERSION" ]; then
+  TF_CUDA_VERSION="8.0"
+  export TF_CUDA_VERSION 
+fi
+write_action_env_to_bazelrc "TF_CUDA_VERSION" "$TF_CUDA_VERSION" 
```

Comment:
Let's also reflect this behavioral change in line 469. The part in the parenthesis can say (use empty to default to CUDA 8.0)